modules/javafx.controls/src/main/java/javafx/scene/control/TreeTableCell.java

```diff
@@ -284,14 +284,24 @@ private void setTreeTableView(TreeTableView<S> value) {
 
     // --- TableRow
     /**
-     * The TreeTableRow that this TreeTableCell currently finds itself placed within.
+     * The {@code TreeTableRow} that this {@code TreeTableCell} currently finds itself placed within.
      */
-    private ReadOnlyObjectWrapper<TreeTableRow<S>> treeTableRow =
-            new ReadOnlyObjectWrapper<TreeTableRow<S>>(this, "treeTableRow");
-    private void setTreeTableRow(TreeTableRow<S> value) { treeTableRow.set(value); }
-    public final TreeTableRow<S> getTreeTableRow() { return treeTableRow.get(); }
-    public final ReadOnlyObjectProperty<TreeTableRow<S>> tableRowProperty() { return treeTableRow;  }
+    private ReadOnlyObjectWrapper<TreeTableRow<S>> tableRow =
+            new ReadOnlyObjectWrapper<TreeTableRow<S>>(this, "tableRow");
+    private void setTableRow(TreeTableRow<S> value) { tableRow.set(value); }
+    public final TreeTableRow<S> getTableRow() { return tableRow.get(); }
```

Comment:
this method is new, so might need a since tag - don't know how to apply it, though (because the property itself is old)

Reply:
I had briefly thought about this, but didn't try it because the docs are copied from the property itself, along with any `@since` tag. I can fix this by copying the docs for the property (which is only a single sentence, so there isn't much duplication), so I'll do that.